components/brave_today/browser/brave_news_controller.cc

```diff
@@ -377,6 +378,7 @@ void BraveNewsController::OnPromotedItemVisit(
 void BraveNewsController::OnSessionCardViewsCountChanged(
     uint16_t cards_viewed_session_total_count) {
   p3a::RecordWeeklyMaxCardViewsCount(prefs_, cards_viewed_session_total_count);
+  p3a::RecordTotalCardViews(prefs_, cards_viewed_session_total_count);
```

Comment:
I assume this is split from `AddLastSessionTotalCardViews` because this is called repeated by js with an updated number, so we want to update the P3A histogram, but not accumulate the number until another session starts so we know the js-side counter has reset?

Please document the rationale in a comment above the two method implementations.